CP-CPS.md

```diff
@@ -0,0 +1,1440 @@
+
+Issues can be filed via the GitHub repository where this document is maintained:
+
+<https://github.com/letsencrypt/cp-cps>
+
+### 1.5.3 Person Determining CP/CPS suitability for the policy
+
+The ISRG PMA is responsible for determining the suitability of this CP/CPS. ISRG policy is informed by results and recommendations received from an independent auditor.
+
+### 1.5.4 CP/CPS approval procedures
+
+The ISRG PMA approves any revisions to this CP/CPS after formal review.
+
+## 1.6 Definitions and Acronyms
+
+### 1.6.1 Definitions
+
+**ACME Protocol**: A protocol used for validation, issuance, and management of certificates. The protocol is an open standard managed by the IETF.
+
+**Baseline Requirements**: A document published by the CAB Forum which outlines minimum requirements for publicly trusted Certificate Authorities.
+
+**CAB Forum**: Certificate Authority / Browser Forum, a group of CAs and browsers which come together to discuss technical and policy issues related to PKI systems. (<https://cabforum.org/>)
+
+**Certificate Repository**: A repository of information about ISRG certificates. It is located at: <https://letsencrypt.org/certificates/>
+
+**Policy and Legal Repository**: A repository of policy and legal documents related to the ISRG PKI. It is located at: <https://letsencrypt.org/repository/>
+
+**Precertificate**: A certificate containing a critical poison extension as defined by RFC 6962 Section 3.1.
+
+**Secure PKI Facilities**: Facilities designed to protect sensitive PKI infrastructure, including CA private keys.
+
+**Trusted Contributor**: A contributor who performs in a Trusted Role. Trusted Contributors may be employees, contractors, or community members. Trusted Contributors must be properly trained and qualified, and have the proper legal obligations in place before performing in a Trusted Role.
+
+**Trusted Role**: A role which qualifies a person to access or modify ISRG PKI systems, infrastructure, and confidential information.
+
+See the Baseline Requirements for additional definitions.
+
+### 1.6.2 Acronyms
+
+| **Acronym** | **Meaning** |
+| --- | --- |
+| ACME | Automated Certificate Management Environment |
+| DV | Domain Validation |
+| HSM | Hardware Security Module |
+| IP | Internet Protocol |
+| ISRG | Internet Security Research Group |
+| PMA | Policy Management Authority |
+| SAN | Subject Alternative Name |
+| TLD | Top Level Domain |
+
+See the Baseline Requirements for additional acronyms.
+
+### 1.6.3 References
+
+[CA/Browser Forum Baseline Requirements for the Issuance and Management of Publicly-Trusted Certificates](https://cabforum.org/baseline-requirements-documents/)
+
+[CA/Browser Forum Network and Certificate System Security Requirements](https://cabforum.org/network-security-requirements/)
+
+### 1.6.4 Conventions
+
+Terms not otherwise defined in this CP/CPS shall be as defined in applicable agreements, user manuals, Certificate Policies, and Certification Practice Statements of the CA.
```

Comment:
Should the references to Certificate Policies and Certificate Practice Statements be removed from this, as this document is the only one of both of those?

Reply:
I think originally I left this in because there might be other applicable CP and/or CPS documents - IdenTrust, for example. We aren't going to depend on term definitions from them though, so that doesn't really matter. It seems fine to leave as a catch all, which is the point of that whole list, but I'd also be fine with removing it.